### PR TITLE
#160617635 User Login feature 

### PR DESCRIPTION
**Pull request by kimbugp**
#### What does this PR do?
- Allow a user to login to the app
#### Description of Task to be completed?
- when a user accesses the URL `/login`, they are allowed to log in to access member features 
#### How should this be manually tested?
- go to the URL https://ah-frontend-lannister-pr-14.herokuapp.com/login
#### What are the relevant pivotal tracker stories?
- [#160617635](https://www.pivotaltracker.com/story/show/160617634)
#### Screenshots (if appropriate)
- login screen
screen shot 2018-11-12 at 16 56 53

- Unsuccessful login
screen shot 2018-11-12 at 16 58 20

- Succesful login
screen shot 2018-11-12 at 16 58 32

### CODE REVIEW

File: src/appUrls/Urls.js [hunk 1]
```diff
@@ -1 +1,3 @@
 export const BASE_URL="https://ah-backend-lannister-staging.herokuapp.com";
+export const LOGIN_URL=`${BASE_URL}/api/users/login/`;
+
```
Comment (LehruAngela) [inline src/appUrls/Urls.js:3]: File naming should be camel case, change Urls to urls.js

File: src/routes/index.js [hunk 2]
```diff
@@ -1,23 +1,29 @@
...
+          <Route path="/login" component={LoginPage} exact strict />
+          <Route path="/register" component={registerUser} exact strict />
+          <Route path="/password_reset" component={ResetRequest} exact strict />
+          <Route path="/reset_confirm/:token" component={NewPassword} strict />
+          <ProtectedRoute
+            path="/create-article"
+            component={CreateArticle}
+            exact
+            strict
+          />
```
Comment (LehruAngela) [inline src/routes/index.js]: Refactor such that the Protected Route is one line to maintain consistency

**Review by LehruAngela — CHANGES_REQUESTED** [review]

File: src/appUrls/Urls.js (hunk 1 above)
Comment (kimbugp) [reply src/appUrls/Urls.js:3]: Noted

File: src/routes/index.js (hunk 2 above)
Comment (kimbugp) [reply src/routes/index.js]: @LehruAngela ,I have made the change

**Pull request merged**